lib/cli/options.ts

```diff
@@ -42,11 +41,11 @@ export class Option {
     if (value && typeof value === 'string') {
       return '' + value;
     }
-    return null;
+    return '';
```

Comment:
This function is supposed to get a "string" value. I'm pretty sure there's other code here that expects a value returned by this function to be a string. I'll have to hunt it down. 

`gulp tsc` emits no errors.